adapter/pokemon_adapter_test.go

```diff
@@ -0,0 +1,126 @@
+package adapter
+
+import (
+	"fmt"
```

Comment:
It's good to use specific testing but you can also add Gomocks library to have more coverage